openstack/loadbalancer/v2/flavors/requests.go

```diff
@@ -0,0 +1,111 @@
+package flavors
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// LIST
+
+type ListOptsBuilder interface {
+	ToFlavorListQuery() (string, error)
+}
+
+type ListOpts struct {
+	Fields []string `q:"fields"`
+}
+
+func (opts ListOpts) ToFlavorListQuery() (string, error) {
+	q, err := gophercloud.BuildQueryString(opts)
+	return q.String(), err
+}
+
+func List(c *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pager {
+	url := rootURL(c)
+	if opts != nil {
+		query, err := opts.ToFlavorListQuery()
+		if err != nil {
+			return pagination.Pager{Err: err}
+		}
+		url += query
+	}
+	return pagination.NewPager(c, url, func(r pagination.PageResult) pagination.Page {
+		return FlavorPage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// CREATE
+
+type CreateOptsBuilder interface {
+	ToFlavorCreateMap() (map[string]interface{}, error)
+}
+
+type CreateOpts struct {
+	Name            string `json:"name,omitempty"`
+	Description     string `json:"description,omitempty"`
+	FlavorProfileId string `json:"flavor_profile_id" required:"true"`
+	Enabled         bool   `json:"enabled,omitempty"`
+}
+
+func (opts CreateOpts) ToFlavorCreateMap() (map[string]interface{}, error) {
+	return gophercloud.BuildRequestBody(opts, "flavor")
+}
+
+func Create(c *gophercloud.ServiceClient, opts CreateOptsBuilder) (r CreateResult) {
+	b, err := opts.ToFlavorCreateMap()
+	if err != nil {
+		r.Err = err
+		return
+	}
+	resp, err := c.Post(rootURL(c), b, &r.Body, nil)
+	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
+	return
+}
+
+// GET
+
+func Get(c *gophercloud.ServiceClient, id string) (r GetResult) {
+	resp, err := c.Get(resourceURL(c, id), &r.Body, nil)
+	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
+	return
+}
+
+// UPDATE
+
+type UpdateOptsBuilder interface {
+	ToFlavorUpdateMap() (map[string]interface{}, error)
+}
+
+type UpdateOpts struct {
+	Name        string `json:"name,omitempty"`
+	Description string `json:"description,omitempty"`
+	Enabled     bool   `json:"enabled"`
```

Comment:
You need to use `omitempty` on `enabled`, otherwise it will set it to false for all update queries if you don't pass it explicitly:
```suggestion
	Enabled     bool   `json:"enabled,omitempty"`
```